### DISCUSSION TITLE
Quasar Browser Cache Issue

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#12967] [CLI - PWA mode] [answered]
Dear Team, 

First of all thanks for a great framework. I am facing one minor issue please help me to fix this.

I have developed an Microservice based enterprise web application by using Quasar (Deployed in Apache Server)  & Springboot (Tomcat). 

When I deploy the frontend application after build it is working fine for some users, but updates are not reflecting for the remaining users it is working after clearing the cache.

Also sometimes I am getting **" chunk not found errors"** but it works after cache clearance & refresh in the chrome browser.

The file 6.b0da48c4.css is available in the server , but it was not getting loaded in browser. Please find screenshot below.

_**Server File**_

![Apache-css-file](https://user-images.githubusercontent.com/8514576/160529426-209e1b12-bb70-47bd-be1c-43d55937c001.JPG)

**_Browser Error_**

![Browser-file](https://user-images.githubusercontent.com/8514576/160529467-2f2646de-77f2-458a-b5f3-d882ad517fbd.JPG)

**Can you  please help me to fix this browser cache & chuck issue**, every deployment I will increase the package.json version still I am facing the issue.

**_Expectation_**

Whenever I deploy the application in production it should reflect for all users without any cache or chuck issue.

**Comment by rstoenescu** [comment]
Make sure that the index.html is also not cached. Otherwise it doesn't matter how well you configure the other assets (js, css, etc).

**Reply by jmriyaz84** [reply]
Thanks Razvan for your quick response . I am not configuring anything for cache , just I am building through npm build & deploying into the production (Apache Server) . Can you please guide / help me how to remove index.html from cache.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@jmriyaz84 

> I am not configuring anything for cache , just I am building through npm build & deploying into the production (Apache Server) 

That's the problem: you have to configure your Apache **Server** to make sure the index.html of your Quasar deployment is not cached.

How to:
https://stackoverflow.com/questions/22891965/how-to-disable-cache-of-apache

Explained why:
https://elelad.medium.com/avoid-cache-trap-when-serving-angular-app-c5981653d156

**Reply by jmriyaz84** [reply]
Thanks @ontwikkelfabriek dobbel , I will try today & let you know the result.
